packages/cozy-harvest-lib/src/datacards/FileDataCard.jsx

```diff
@@ -174,23 +176,58 @@ const makeQueryFromProps = ({ accountId, trigger }) => ({
     .indexFields(['dir_id', 'cozyMetadata.createdAt'])
     .sortBy([{ dir_id: 'desc' }, { 'cozyMetadata.createdAt': 'desc' }])
     .limitBy(5),
-  as: `fileDataCard_io.cozy.accounts/${accountId}/io.cozy.files`,
+  as: `fileDataCard_io.cozy.files/${trigger.message.folder_to_save}/io.cozy.files`,
   fetchPolicy: CozyClient.fetchPolicies.olderThan(30 * 1000)
 })
 
-const FileDataCard = ({ filesCol, konnector, trigger }) => {
-  const { data: files } = filesCol
+const makeSourceAccountQueryFromProps = ({ accountId }) => ({
+  query: Q('io.cozy.files')
+    .where({
+      'cozyMetadata.sourceAccount': accountId,
```

Comment:
~~This should be the only method used, no?~~ explained in the PR description. 

nit: adding the info in the commit message or in a comment can be easier to understand

Reply:
Added in the description of the commit.